views/content/revision.html

```diff
@@ -11,28 +11,30 @@
 {% block main %}
 <h1 class="base--h1">{{title}}</h1>
 
-<h2 class="base--h2">Revisions</h2>
-
 {% if content %}
 
-  <table class="table--style">
-    <tr class="table--tr">
-      <th class="table--th">revision</th>
-      <th class="table--th">{{config.base}}</th>
-      <th class="table--th">{{config.actions.edit}}</th>
-      <th class="table--th">date</th>
+<table class="base--table">
```

Comment:
so....here's a weird quirk....

(tl;dr: please delete this file in this PR)

Due to the need to create routes, this template actually doesn't get used. It's detritus I shoulda deleted. All routes have to be covered or you get bugs, and I needed a route for the actual revision number in there. So, this revision route uses the same template (views/content/content.html) as the edit page.

The templates:

url: /content ([staging](https://watson-cms.stage1.mybluemix.net/content/))
template: /views/content/home ([line in content routes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/content.js#L38))

url: /content/example ([staging](https://watson-cms.stage1.mybluemix.net/content/example/))
template: /views/content/landing ([line in content routes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/content.js#L72))

url: /content/example/add ([staging](https://watson-cms.stage1.mybluemix.net/content/example/add))
template: /views/content/add ([line in content routes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/content.js#L113))

url: /content/example/longUUID ([staging](https://watson-cms.stage1.mybluemix.net/content/example/b45fca3d-63dd-4a32-84e0-3e66637caae3))
template: /views/content/content ([line in content routes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/content.js#L162))

url: /content/example/longUUID ([staging](https://watson-cms.stage1.mybluemix.net/content/example/b45fca3d-63dd-4a32-84e0-3e66637caae3))
template: /views/content/content ([line in content routes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/content.js#L162))

*note: quirk* url: /content/example/longUUID/revNum ([staging](https://watson-cms.stage1.mybluemix.net/content/example/b45fca3d-63dd-4a32-84e0-3e66637caae3/39))
template: /views/content/content ([line in content routes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/content.js#L214))

url: /content/example/longUUID/revNum/edit ([staging](https://watson-cms.stage1.mybluemix.net/content/example/b45fca3d-63dd-4a32-84e0-3e66637caae3/39/edit))
template: /views/content/add ([line in content routes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/content.js#L282))

url: /content/example/longUUID/revNum/approve ([staging](https://watson-cms.stage1.mybluemix.net/content/example/b45fca3d-63dd-4a32-84e0-3e66637caae3/39/approve))
template: /views/content/approval ([line in workflows routes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/workflows.js#L112))